src/main/kotlin/com/broeskamp/monorepo/gradle/plugin/android/AndroidExtension.kt

```diff
@@ -3,13 +3,8 @@ package com.broeskamp.monorepo.gradle.plugin.android
 import org.gradle.api.Project
 import org.gradle.api.provider.Property
 import org.gradle.kotlin.dsl.property
-import org.jetbrains.kotlin.gradle.plugin.extraProperties
 
 open class AndroidExtension(val project: Project) {
-  fun setAndroidDefaultProperties() {
-    project.extraProperties.set("android.useAndroidX", "true")
-    project.extraProperties.set("android.nonTransitiveRClass", "true")
```

Comment:
Warum wollen wir das nicht, das klingt gut?

Reply:
Das wollen wir, ja. Aber das Android Plugin nimmt das nicht an. Muss in der Gradle.properties eingestellt werden. Deshalb weg